src/app/derive/derive.component.html

```diff
@@ -1,117 +1,82 @@
-      <div class="fs-18px mt-30px">
-        <span class="fs-underline"
-          ><b
-            >You're about to grant {{ globalVars.hostname }} with signing
-            access:</b
-          ></span
-        ><br />
-        This means <b>{{ globalVars.hostname }}</b> will be able to make
-        transactions on your behalf as if it knew your DeSo seed. This
-        permission can be revoked anytime and it will automatically expire in
-        about {{ expirationDays }} days.
-      </div>
-    </div>
-    <div *ngIf="globalVars.callback">
-      <div class="fs-18px mt-30px">
-        <span class="fs-underline"
-          ><b>You're about to grant signing access:</b></span
-        ><br />
-        This means the mobile application will be able to sign transactions on
-        your behalf as if it knew your DeSo seed. This permission can be revoked
-        anytime and it will automatically expire in about
-        {{ expirationDays }} days. <br /><br />
-        <span class="fs-underline"><b>DISCLAIMER:</b></span
-        ><br />
-        <b
-          >This session is only for mobile applications. If you are not using a
-          mobile application, you should exit this window.</b
-        >
-      </div>
-    </div>
-
-    <div *ngIf="publicKeyBase58Check" class="pt-15px">
-      <button
-        type="button"
-        class="button button-small button-primary"
-        (click)="approveDerivedKey()"
-      >
-        Approve
-      </button>
-    </div>
-    <div class="row" *ngIf="!publicKeyBase58Check">
-      <div class="d-flex flex-column col-sm-6" *ngIf="hasUsers">
-        <ul class="list-group mt-7px mb-30px">
-          <li
-            *ngFor="let item of allUsers | keyvalue"
-            class="list-group-item list-group-item-action cursor-pointer"
-            (click)="selectAccountAndDeriveKey(item.key)"
-          >
-            <div class="w-100">
-              <div *ngIf="!item.value.username" class="text-truncate">
-                {{ item.key }}&hellip;
-              </div>
-              <div
-                *ngIf="item.value.username"
-                class="d-flex align-items-center"
-              >
-                <div class="avatar mr-15px" [appAvatar]="item.key"></div>
-                <div class="text-truncate">{{ item.value.username }}</div>
-              </div>
-            </div>
-          </li>
-        </ul>
-      </div>
-      <div
-        class="d-flex align-items-center flex-column"
-        [class.col-sm-6]="hasUsers"
-        [class.col-sm-12]="!hasUsers"
-      >
-        <button
-          class="btn btn-default log-in-google mb-30px"
-          (click)="launchGoogle()"
-          *ngIf="!globalVars.hideGoogle"
-        ></button>
-        <button
-          class="btn btn-default log-in-seed"
-          (click)="redirectLoadSeed()"
-        ></button>
-        <button
-          class="btn btn-default sign-up-seed"
-          (click)="redirectSignUp()"
-        ></button>
```

Comment:
how are we handling the situation where a user isn't logged in to any accounts now? We need some path for this.

Reply:
Wouldn't they just login from where they were before?